targets/PythonSdk/source/setup.py

```diff
@@ -0,0 +1,20 @@
+import setuptools
+with open("README.md", "r") as fh:
+        long_description = fh.read()
+        
+setuptools.setup(
+    name="playfab",
+    version="0.0.1",
+    author="Todd Bello",
+    author_email="toddbell@microsoft.com",
```

Comment:
This will get you a large number of direct emails from customers.
Generally we don't put an author, and add information directing them to the forums.
https://community.playfab.com/questions/ask.html